src/desktops/dskHostGame.cpp

```diff
@@ -431,6 +431,7 @@ void dskHostGame::UpdatePlayerRow(const unsigned row)
  */
 void dskHostGame::Msg_PaintBefore()
 {
+    Desktop::Msg_PaintBefore();

```

Comment:
evtl sollten wir allgemein eine art typedef base in den klassen machen, dann kann man immer "base::..." schreiben?